metapool/sample_sheet.py

```diff
@@ -645,166 +985,49 @@ def make_sample_sheet(metadata, table, sequencer, lanes, strict=True):
     SampleSheetError
         If one of the required columns is missing.
     """
-    messages = _validate_sample_sheet_metadata(metadata)
+    required_attributes = ['SheetType', 'SheetVersion', 'Assay']
+
+    for attribute in required_attributes:
+        if attribute not in metadata:
+            raise ValueError("'%s' is not defined in metadata" % attribute)
+
+    sheet_type = metadata['SheetType']
+    sheet_version = metadata['SheetVersion']
+    assay_type = metadata['Assay']
+
+    if sheet_type == _STANDARD_SHEET_TYPE:
+        if assay_type == _AMPLICON:
+            sheet = AmpliconSampleSheet()
+        elif assay_type == _METAGENOMIC:
+            if sheet_version == '90':
+                sheet = MetagenomicSampleSheetv90()
+            elif sheet_version in ['95', '99', '100']:
+                # 95, 99, and v100 are functionally the same type.
+                sheet = MetagenomicSampleSheetv100()
```

Comment:
Maybe I'm missing something but I don't understand why do we need 95, 99 and 100 if they are the same ...